### DISCUSSION TITLE
Spinner bonus still awards 50 per tick while "Classic" mod is applied

### DISCUSSION THREAD

**Discussion by roxandtol** [ppy/osu#18680] [Q&A]
### Type

Game behaviour

### Bug description

When you are spinning a spinner and you finish it, you get a bonus. On Lazer that bonus is 50x, and in stable, it is 1000x, so when you have applied the classic mode, that is supposed to bring stable behavior back, you still get the 50x on spinners.

### Screenshots or videos

![Screenshot_2022-06-13-23-46-29-76_25a1a32208bbcdc1d450b7aa854bb161](https://user-images.githubusercontent.com/41841903/173451193-c5be43f4-6c93-402e-a3ce-092d91b463d8.jpg)

### Version

2022.612.0

### Logs

[database.log](https://github.com/ppy/osu/files/8894763/database.log)
[performance.log](https://github.com/ppy/osu/files/8894764/performance.log)
[network.log](https://github.com/ppy/osu/files/8894765/network.log)
[runtime.log](https://github.com/ppy/osu/files/8894766/runtime.log)

**Comment by frenzibyte** [comment]
Moving this to discussion since "Classic" mods generally don't affect the score hit objects award.

**Comment by willyosu** [comment]
There may be some confusion here between the "Classic" mod and "Classic" scoring. The mod itself doesn't change how scoring works (everything follows standardized scoring), the only way to emulate stable's scoring is changing the scoring display mode in the settings.

That being said the spinner bonus in lazer does display and apply a 50x instead of 1000x regardless of which scoring display mode is used. It would be nice to have the old bonus apply when using the other scoring display mode, but given different bonuses potentially changing the order of leaderboards compared to standardized scoring I'm not sure that it's feasible. Maybe there's some smart people out there with a way to make that work.

Apart from the actual bonus, maybe switching between standardized and classic modes could change the displayed spinner bonus value as shown in the screenshot between 50x and 1000x visually, but there's really not much reason to only have it look that way when the behavior doesn't match.

**Reply by bdach** [reply]
> given different bonuses potentially changing the order of leaderboards compared to standardized scoring I'm not sure that it's feasible

This is already a problem. Because the spinner bonus is a flat 50 points, it ends up being more impactful in standardised scoring, and will reorder scores.

**Comment by WitherFlower** [comment]
A solution would be to show how much the score increased with the spinner when using classic scoring. This would not be a constant amount (due to the quadratic nature of classic scoring), but it would make more sense than showing +50

**Comment by WitherFlower** [comment]
Solution proposed in https://github.com/ppy/osu/pull/34390
